### PR TITLE
Couple tiny fixes

### LINKED ISSUE #13274: VideoTexture leaves camera stream open after being disposed
I noticed an issue where when you create a VideoTexture using the VideoTexture.CreateFromWebCamAsync helper function the camera stream still remains open for a while after the VideoTexture has been disposed. Looking at the code the issue is that the VideoTexture helper is creating a MedaiStream object to pull the camera frames but nothing is calling `stop()` on the MediaStream tracks. That leaves the camera stream open until garbage collection disposes the orphaned MediaStream object.

**Repro**
Repro playground: https://playground.babylonjs.com/#EWQ1UH#6

The comment block at the top of the playground describes the issue, expected results, and a proposed solution.

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser Edge/Chrome

### PR DESCRIPTION
**Pull request by sebavan**
Fix #13274

### CODE REVIEW

**Review by Popov72 — APPROVED** [review]

**Comment by RaananW** [comment]
Seems like the newly introduced PBR tests are failing?

File: packages/dev/core/src/Materials/Textures/videoTexture.ts [hunk 1]
```diff
@@ -461,6 +461,9 @@ export class VideoTexture extends Texture {
                     });
                 }
                 videoTexture.onDisposeObservable.addOnce(() => {
+                    stream.getTracks().forEach((track) => {
```
Comment (CoPrez) [inline packages/dev/core/src/Materials/Textures/videoTexture.ts:464]: The WebAPI doesn't make it super clear, but from my understanding of the intent. Whoever creates the stream (via `navigator.getUserMedia()`) should be the one to close it.

In this helper function we don't know if the user created the stream and passed it into CreateFromStreamAsync or if VideoTexture created it internally via CreateFromWebCamAsync (which then calls CreateFromStreamAsync). In the former case it should be up to the caller to close the stream, and in the latter case up to VideoTexture.

**Comment by bghgary** [comment]
Disabling auto-merge because of @CoPrez's comment

File: packages/dev/core/src/Materials/Textures/videoTexture.ts (hunk 1 above)
Comment (CoPrez) [reply packages/dev/core/src/Materials/Textures/videoTexture.ts:464]: I would propose capturing the VideoTexture returned from `CreateFromStreamAsync` inside of `CreateFromWebCamAsync` and then adding on another onDisposed handler that closes the stream created by `CreateFromWebCamAsync`.

The reason for the subtle difference is that the caller could be using the MediaStream in multiple instances or outside of Babylon as well. Closing the stream closes it across all instances so you should only do it if you're the owner of the stream.

**Pull request merged**